tests/ui/codegen/large-stack-size-issue-83060.rs

```diff
@@ -0,0 +1,19 @@
+// This test checks that allocating a stack size of 1GB or more results in a warning
+//@build-pass
+
```

Comment:
Try this:
```suggestion
//@ only-64bit

```

Reply:
Thank you! Did not see the diff until after I added your suggestion from Zulip message!